src/framework/theme/components/stepper/step.component.ts

```diff
@@ -102,4 +102,11 @@ export class NbStepComponent {
       this.stepControl.reset();
     }
   }
+
+  /**
+   * Mark step as interacted
+   */
+  visit() {
+    this.interacted = true;
```

Comment:
I think it would be slightly clearer to name this variable as `visited`. What do you think?